### PR TITLE
Create Binary_Tree_sum_of_nodes_at_kth_level.md

### LINKED ISSUE #8705: Binary Tree : Sum of nodes at kth level
### Description

Documentation on how to find the sum of nodes at kth level in a binary tree

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by saloni33**
<hr>

## Description 📜

Documentation for sum of all nodes present at kth level in binary tree
Fixes  #8705 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by saloni33** [comment]
@vasu-1 and @Suvraneel  please review

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@saloni33 
- Add a link of your file to the readme file of the folder 
- Please mention the approach/algorithm of your code
- Avoid using html tags as done in lines 5-9

**Comment by saloni33** [comment]
@apeksha235  can you tell me what to use instead of &nbsp; . I am unable to find another method.

**Comment by saloni33** [comment]
@apeksha235  made the changes can you review again

File: C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Sum of all nodes at Kth level in Binary Tree - <br/>
+Consider a binary tree as -  <br/>
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md:2]:
```suggestion
Consider a binary tree as -   
```

Avoid using HTML tags in md unless inevitable cz they tend to break
Convention for line break is 2 spaces at end of the line

File: C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          /   \
+        3      7
+       / \    /
+      2   4  6
+```
+
+And we need to find the sum of all nodes present at level 2nd. <br/>
+The output will be  - 12  <br/>
+
+## Approach <br/>
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md:15]: Ditto
Also, br not required in headings
```suggestion
## Approach  
```

File: C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+- Traverse the binary tree in Level order manner.
+- During traversal, pop each element out of the queue and push its left and right child (if present in given tree).
+- Keep the record of current level in binary tree.
+- To keep the record of current level, maintain a variable *level* and increase it whenever a child is traversed from the parent.
+- When the *level* becomes equal to k (as given in question), pop the elements from the queue and calculate their sum.
+
+## Code:  <br/>
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md:22]: "

File: C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+```
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md:24]:
```suggestion
```cpp
```
Use code highlighting

File: C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+## Output : <br/>
+12 <br/>
+
+## Time Complexity: <br/>
+O(n) where n is the number of nodes in the binary tree  <br/>
+
+## Space Complexity: <br/>
+O(n) where n is the number of nodes in the binary tree
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Binary Tree/Binary_Tree_sum_of_nodes_at_kth_level.md:121]: All br s to be removed

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@saloni33  Good content

**Comment by saloni33** [comment]
@Suvraneel sir done all the required changes

**Comment by Suvraneel** [comment]
@saloni33  GG 💯  Noice work 🚀 ❤️ :octocat:  LGTM

**Pull request merged**